nimble.nimble

```diff
@@ -1,7 +1,9 @@
-import ospaths
-template thisModuleFile: string = instantiationInfo(fullPaths = true).filename
+import strutils
 
-when fileExists(thisModuleFile.parentDir / "src/nimblepkg/common.nim"):
+const
+  parentDir = currentSourcePath.rsplit(seps={'/', '\\', ':'}, maxsplit=1)[0]
```

Comment:
Why is ':' listed here?

Reply:
Poor man's path split. Will all go away since @dom96 wants to hard code the version. This is causing another issue when nimble is installed as a package.

Reply:
os.nim cannot be imported in nimscript.

Reply:
It can with Nim devel and before that there was ospaths.nim

Reply:
Removed all this code and replaced version = "0.9.0".